app/styles/pages/course-admin-submission-diff.scss

```diff
@@ -0,0 +1,7 @@
+.course-admin-submission-diff-container {
+  code-mirror {
+    .cm-collapsedLines {
+      margin-left: -24px;
```

Comment:
@VasylMarchuk I don't think this is actually being applied - the bar still has the  huge left padding: 

<img width="1114" alt="Screenshot 2024-10-15 at 16 58 41" src="https://github.com/user-attachments/assets/c22bfb45-bd2c-4081-b5e6-368e3d118db6">

Reply:
Fixed now. Rewrote the rule in CSS and it is working. Strangely enough, it worked in the SCSS form on my localhost, but not after being deployed to prod.